x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -207,27 +177,51 @@ export const CaseView = React.memo(({ caseId, subCaseId, userCanCrud }: Props) =
       {casesUi.getCaseView({
         allCasesNavigation: {
           href: formattedAllCasesLink,
-          onClick: backToAllCasesOnClick,
+          onClick: async (e) => {
+            if (e) {
+              e.preventDefault();
+            }
+            return navigateToApp(`${APP_ID}:${SecurityPageName.case}`, {
```

Comment:
Seems like `navigateToApp` returns `void` do we need the `return` here?

Reply:
yes its returning the `Promise<void>`. not a ts error but a warning